src/TBuffer.cpp

```diff
@@ -2253,6 +2253,64 @@ void TBuffer::append(const QString& text, int sub_start, int sub_end, TChar form
     }
 }
 
+// Wraps text to max line length of mWrapAt
+// Applies indentation of mWrapIndent to wrapped lines
+QString TBuffer::wrapText(const QString& text) const
+{
+    if (mWrapAt <= mWrapIndent) {
+        qWarning() << "mWrapAt (" << mWrapAt << ") is too small to accommodate mWrapIndent (" << mWrapIndent << ")";
+    }
+
+    const QString wordBreaks = qsl(",.- ");
+    QString wrappedText;
+    QString currentWord;
+    QString currentLine;
+    int wordsInCurrentLine = 0;
+
+    for (int i = 0; i < text.size(); i++) {
+        bool const at_newline = text.at(i) == '\n';
```

Comment:
Some `QChar`s are special cased and are available directly (see https://doc.qt.io/qt-5/qchar.html#SpecialCharacter-enum ) so that you do not need to have a C raw-character which would trigger the rejection (at compilation) of raw-string/raw-character that `QT_NO_CAST_FROM_ASCII` / `QT_NO_CAST_TO_ASCII` produce if they are ever `#define`d.

Also, for consistency, put the `const` before the type specifier:
```suggestion
        const bool at_newline = (text.at(i) == QChar::LineFeed);
```